easel/test/screens/creator_hub/creator_hub_screen_test.dart

```diff
@@ -0,0 +1,109 @@
+import 'package:easel_flutter/easel_provider.dart';
+import 'package:easel_flutter/screens/creator_hub/creator_hub_screen.dart';
+import 'package:easel_flutter/screens/creator_hub/creator_hub_view_model.dart';
+import 'package:easel_flutter/utils/constants.dart';
+import 'package:easel_flutter/utils/easel_app_theme.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:get_it/get_it.dart';
+import 'package:provider/provider.dart';
+
+import '../../extensions/size_extension.dart';
+import '../../mock/media_info.mocks.dart';
+import '../../mock/mock_audio_imp.dart';
+import '../../mock/mock_constants.dart';
+import '../../mock/mock_file_utils.dart';
+import '../../mock/mock_repository.dart';
+import '../../mock/mock_video_player.dart';
+
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
+  final repository = MockRepositoryImp();
+  final creatorHubViewModel = CreatorHubViewModel(repository);
+  GetIt.I.registerLazySingleton(() => creatorHubViewModel);
+
+  final fileUtils = MockFileUtilsHelperImp();
+  final videoPlayerHelper = MockVideoPlayerHelperImp();
+  final audioPlayerHelper = MockAudioPlayerHelperImp();
+  final easelProvider = EaselProvider(
+    repository: repository,
+    fileUtilsHelper: fileUtils,
+    audioPlayerHelperForFile: audioPlayerHelper,
+    audioPlayerHelperForUrl: audioPlayerHelper,
+    mediaInfo: MockMediaInfo(),
+    videoPlayerHelper: videoPlayerHelper,
+  );
+  GetIt.I.registerLazySingleton(() => easelProvider);
+
+  group("Creator Hub Screen Test Cases", () {
```

Comment:
The purpose of group is to contain the common code which i dont see you are using it for.